src/shared/components/tracks/PtmAnnotationTable.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import {PostTranslationalModification} from "shared/api/generated/GenomeNexusAPI";
+import LazyMobXTable, {Column} from "shared/components/lazyMobXTable/LazyMobXTable";
+import DefaultTooltip from "shared/components/defaultTooltip/DefaultTooltip";
+
+export interface IPtmSummaryTableProps
+{
+    data: PostTranslationalModification[];
+    columns?: Column<PostTranslationalModification>[];
+    initialSortColumn?: string;
+    initialSortDirection?: 'asc'|'desc';
+    initialItemsPerPage?: number;
+}
+
+// LazyMobXTable is a generic component which requires data type argument
+class PtmTable extends LazyMobXTable<PostTranslationalModification> {}
+
+export default class PtmAnnotationTable extends React.Component<IPtmSummaryTableProps, {}>
+{
+    public static defaultProps = {
+        data: [],
+        columns: [
+            {
+                name: "Position",
+                order: 1.00,
+                render: (d: PostTranslationalModification) => (<div style={{textAlign: "right"}}>{d.position}</div>),
+                sortBy: (d: PostTranslationalModification) => d.position
+            },
+            {
+                name: "Type",
+                order: 2.00,
+                render: (d: PostTranslationalModification) => <div>{d.type}</div>,
+                sortBy: (d: PostTranslationalModification) => d.type
+            },
+            {
+                name: "Citation",
+                order: 3.00,
+                render: (d: PostTranslationalModification) => (
+                    <DefaultTooltip
+                        placement="right"
+                        overlay={() => <div style={{maxWidth: 300}}>{d.pubmedIds.join(", ")}</div>}
```

Comment:
Here, we should use the `ReferenceList` component introduced by #2097 (which is not merged yet)